sandbox.html

```diff
@@ -231,7 +231,8 @@
       fullGeometries,
       zoomToGeometry,
       distance,
-      lengthUnit
+      lengthUnit,
+      additionalLayerDefinitions
```

Comment:
These layers are imported in the background of the drawing for reference.
For the param name we thought of these possibilities:
- additionalLayerDefinitions
- referenceLayerDefinitions
- adhocLayerDefinitions